requirements.txt

```diff
@@ -34,3 +34,4 @@ tooz>=1.58.0 # Apache-2.0
 WebOb>=1.7.1 # MIT
 pysnmp>=4.4.11 # BSD
 redis>=3.3.8 # MIT
+pyopenssl==19.1.0 # Apache-2.0
```

Comment:
It seems that we haven't import it, do we still need it?

Reply:
Removed

Reply:
?? still here?